module/checkbox.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable import/no-mutable-exports */
+
+import { bookList, Book } from '../src/index.js';
+
+let notCompleted = [];
+let filtered = [];
+function clearAllTask() {
+  notCompleted = [];
+  filtered = bookList.filter((book) => book.completed !== true);
+  for (let i = 0; i < filtered.length; i += 1) {
+    const newBook = new Book(filtered[i].description, filtered[i].completed, i + 1);
+    notCompleted.push(newBook);
+  }
+}
+export { clearAllTask, notCompleted };
```

Comment:
Kindly consider exporting with the method that you write the `export` code next to the function or variable that you want to import example: `export let not completed = [];` and instead of exporting a value from a function you can use return.

Reply:
THanks the change was implemented.